protocol/app/app.go

```diff
@@ -747,8 +747,8 @@ func New(
 		tkeys[statsmoduletypes.TransientStoreKey],
 		// set the governance and delaymsg module accounts as the authority for conducting upgrades
 		[]string{
-			authtypes.NewModuleAddress(delaymsgmoduletypes.ModuleName).String(),
-			authtypes.NewModuleAddress(govtypes.ModuleName).String(),
+			delaymsgmoduletypes.ModuleAddress.String(),
```

Comment:
nit: here and below - can we order these to be gov and then delay to be consistent?